app/assets/javascripts/Resources/coupon.coffee

```diff
@@ -0,0 +1,13 @@
+app = angular.module('customerApp')
+
+app.factory('Coupon', ['$resource', ($resource) ->
```

Comment:
why is the folder name ```Resources```?

Reply:
cause @gsnovak just can't let go of his .net ways